docs/guides/avoid_blocking_playwright_fingerprints_off.py

```diff
@@ -0,0 +1,16 @@
+from crawlee.browsers import BrowserPool
+from crawlee.playwright_crawler import PlaywrightCrawler
+
+# Create a browser pool with use_fingerprints set to False
+browser_pool = BrowserPool.with_default_plugin(
+    headless=True,
+    kwargs={
+        'use_fingerprints': False,
+    },
+)
```

Comment:
Additional kwargs can be provided directly.

```suggestion
browser_pool = BrowserPool.with_default_plugin(
    headless=True,
    use_fingerprints=False,
)
```

Reply:
for this it's not working directly so I added the extra code in the conversation as well and then there's a new issue , I have attached screenshots for it

Reply:
Yes, this is not going to work, since `use_fingerprints` is the parameter of the `Plugin` and not the `BrowserPool`.